store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/integration/AccumuloAggregationIT.java

```diff
@@ -239,10 +235,8 @@ public void shouldHandleAggregationWhenGroupByPropertiesAreNull() throws Operati
         final Entity expectedEntity = new Entity.Builder()
                 .vertex(VERTEX)
                 .group(TestGroups.ENTITY)
-                .property(AccumuloPropertyNames.COLUMN_QUALIFIER, ",") //String Aggregation is combining two empty strings -> "",""
```

Comment:
What happens now in this scenario ? Does entity1 need to be modified for testing the comments?

Reply:
Because they were added as `null` it is the same as not adding them at all. I think this behaviour is okay.